tslint.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
if we have tslint, should we remove eslint?

Reply:
We can keep both for now as the project has both plain javascript and will incrementally grow in typescript:
> When it comes to linting TypeScript code, there are two major linting options to choose from: TSLint and ESLint. TSLint is a linter that can only be used for TypeScript, while ESLint supports both JavaScript and TypeScript.